packages/@okta/vuepress-site/docs/guides/device-authorization-grant/main/index.md

```diff
@@ -0,0 +1,154 @@
+* `verification_uri`: The URL that the user needs to enter into their phone to start signing in.
+* `expires_in`: The number of seconds that this set of values is valid. After the device code and user code expire, the user has to start the device verification process over.
+* `interval`: The number of seconds that the device should wait between polling to see if the user has finished signing in.
+
+The `user_code` and `verification_uri` must appear on the smart device for the user.
+
+> **Note:** A QR code compliant `verification_uri` is coming soon.
+
+#### Example of the display on the smart device
+
+![Verification on the smart device:](/img/DeviceAuthGrant2.png)
+
+### Request for user access, ID, and refresh tokens
+
+To retrieve tokens for the user, the smart device needs to make a request to the `/token` endpoint.
+
+**Request example**
+
+```bash
+  curl --location --request POST 'https://${yourOktaDomain}/oauth2/default/v1/token' \
+  --header 'Content-Type: application/x-www-form-urlencoded' \
+  --data-urlencode 'client_id={clientId}' \
+  --data-urlencode 'grant_type=urn:ietf:params:oauth:grant-type:device_code' \
+  --data-urlencode 'device_code={deviceCode)' \
+```
+
+Note the paramters that are being passed:
+
+* `grant_type`: Identifies the mechanism that Okta uses to retrieve the tokens. Value: `urn:ietf:params:oauth:grant-type:device_code`
+* `device_code`: The string that the device uses to exchange for an access token. Use the `device_code` value from the device verification response.
+
+**Response example**
+
+Okta returns a pending response if the user doesn't complete the authentication.
+
+```json
+  {
+     "error": "authorization_pending",
+     "error_description": "The device authorization is pending. Please try again later."
+  }
+```
+
+After the user completes the authentication process by visiting the `/activate` URL and follow the instructions on their device to get an activation code, the token is returned in the response from the `/token` endpoint.
+
+```json
+ {
+   "token_type": "Bearer",
+   "expires_in": 3600,
+   "access_token":
+      "eyJraWQ...JQuDJh8g",
+   "scope": "offline_access openid profile",
+   "refresh_token": "zcLdr1FBXwtI9ej98VVVwtjDd-SmaoL06qr_UcY2tNA",
+   "id_token": "eyJraWQ...WI6KR0aQ"
+ }
+```
+
+If you need a refresh token, ensure that you've initially requested the `offline_access` scope through the `/device/authorize` endpoint. To renew access tokens in the future, you can use the refresh token.
+
+> **Note:** The unique user code is valid for 10 minutes. The user needs to use this code and complete the authentication process within that time. If not, the unique user code expires and the device can request a new user code by using the `/device/authorize` endpoint.
+
+## Revoke the token
+
+To revoke the tokens, the smart device must make a request to the `/revoke` endpoint.
+
+```bash
```

Comment:
Do we not show any info about the response when the response does not contain a body?

Reply:
@larsjohansen-okta - not usually - but we could at least say the response returns a 200 OK?

Reply:
We should be consistent with what we do here: https://github.com/okta/okta-developer-docs/pull/2398#discussion_r682279529

Reply:
Fixed.